### ISSUE TITLE
Some minor typos and errors

### ISSUE THREAD

**Issue by jip** [IDNI/parser#7]
1. REPL: the shortcut `n` of command `grammar` is not recognized:

```console
> cat SSS.tgf
@use char classes alpha.
@start S.
S => S S S | S S | 'a'.

> tgf SSS.tgf repl
[ "SSS.tgf" S ] tgf> v
TGF version: 24e52d2
[ "SSS.tgf" S ] tgf> n
invalid command: Syntax Error: Unexpected 'n' at 1:1 (1): n
[ "SSS.tgf" S ] tgf> grammar
grammar:
@use char classes alpha.
@disable disambig.
@start S.
S => S S S | S S | 'a'.
[ "SSS.tgf" S ] tgf> quit
Quit.
```

2. The script `examples/tml_parser/gen` refers to obsoleted `parser_gen` utility:

```bash
../../build-Release/parser_gen tml_parser ./tml.tgf > ./tml_parser.generated.h
```
should be:
```bash
../../build-Release/tgf ./tml.tgf gen --name tml_parser --output ./tml_parser.generated.h
```
instead.

3. The grammar `examples/tml_parser/tml.tgf` refers to obsoleted `use_char_class` directive:

```grammar
@use_char_class eof, space, digit, xdigit, alpha, alnum, punct, printable.
```
should be:
```grammar
@use char class eof, space, digit, xdigit, alpha, alnum, punct, printable.
```
instead.

4. It would be nice to instruct the `src/tree/Makefile` script to produce `tree` unility not `a.out`:

```makefile
	c++ main.cpp vec.cpp -g -std=c++20
```
should be:
```
	c++ main.cpp vec.cpp -g -std=c++20 -o tree
```
instead.

**Comment by tklip** [comment]
Hi @jip, thanks for your comments.
1. short for `grammar` command should be `g`, not `n`
2. and 3. are not updated since it is a grammar and a parser of Tau Meta Language. Development of TML is stalled now since we focus on finishing Tau language first - this example was used for developing the parser but it would be better to remove it from the parser repository and add it to the repository of TML when we continue to work on it.
4. src/tree contains a code to be incorporated into parser. It's not a part of parser yet. When it will be it would be covered by cmake as the rest of the codebase.

**Comment by jip** [comment]
Thanks for reply.
1. Then [the help in tools/tgf tgf_cli.cpp](https://github.com/IDNI/parser/blob/e2defd23cd032a83e69f464ebb59f991027b57cd/tools/tgf/tgf_cli.cpp#L838) should be fixed.
2. and 3. Ok.

**Comment by tklip** [comment]
I see. Thanks. fixed in d1b048e
